.github/workflows/python2-3-app.yml

```diff
@@ -0,0 +1,43 @@
+name: DB12 testing
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 2.7]
+        
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      
+    - name: Add conda to system path
+      run: |
+        echo $CONDA/bin >> $GITHUB_PATH
+        
+    - name: Set up Conda and Install dependencies
+      run: |
+        conda create -c conda-forge -c free -n python python=${{ matrix.python-version }}
+        conda info --env
+        source /usr/share/miniconda/etc/profile.d/conda.sh
+        conda activate python
+        pip install pytest
+        pip install future
+        pip install pylint
```

Comment:
Currently, there are only 3 dependencies so this is probably fine, but you could also add them in a `requirements.txt` file and make a `pip install -r requirements.txt` instead of calling `pip install <package>`.
Then another option, which is not mandatory, is to add the `requirements.txt` file to a conda `environment.yml` file. 

- `requirements.txt` example in DIRAC: https://github.com/DIRACGrid/DIRAC/blob/integration/requirements.txt
- an option to add `requirements.txt` in `environment.yml`: https://stackoverflow.com/questions/35245401/combining-conda-environment-yml-with-pip-requirements-txt
- an `environment.yml` example in DIRAC:  https://github.com/DIRACGrid/DIRAC/blob/integration/environment.yml
- how to call `environment.yml` in DIRAC: https://github.com/DIRACGrid/DIRAC/blob/integration/.github/workflows/basic.yml#L38